sdk/vision/ci.yml

```diff
@@ -0,0 +1,31 @@
+trigger:
```

Comment:
Doesn't need to be for this PR, but are you planning on adding live tests / tests.yml as well?

Reply:
It's not planned for this PR as we are in a rush to get a preview package out in January. Certainly something we need to consider as we prioritize work items for the next preview update.